hummingbot/wallet/ethereum/watcher/new_blocks_watcher.py

```diff
@@ -107,7 +107,7 @@ def block_number(self) -> int:
                         incoming_block: AttributeDict = await self.call_async(
                             functools.partial(
                                 self._w3.eth.getBlock,
-                                self._block_number_to_fetch,
+                                'latest',
```

Comment:
if we fetch only the latest block theres a chance we might skip blocks?

Reply:
fixed.